tests/musig_wasm_tests.rs

```diff
@@ -0,0 +1,188 @@
+#[cfg(test)]
```

Comment:
This may be overkill, but since this code is not autogenerated there could be mistakes and I would prefer to have functional test for it. Also, I'll add more tests. I even have JS test which is gitignored, though I should probably push it